templates/mood.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html>
+<head><meta charset="utf-8"><title>Mood: {{ mood }}</title></head>
```

Comment:
Consistent formatting pls 

```suggestion
<head>
  <meta charset="utf-8">
  <title>Mood: {{ mood }}</title>
</head>
```

Exista formattere si de HTML/Jinja templates in VSCode